plugins/catalog/src/js/catalog/__tests__/PackageDetailTab-test.js

```diff
@@ -1,8 +1,8 @@
-const packageDescribeFixtures = require("../../../../../tests/_fixtures/cosmos/package-describe.json")
+const packageDescribeFixtures = require("../../../../../../tests/_fixtures/cosmos/package-describe.json")
```

Comment:
Can't we do "#SRC/../tests" ?

Reply:
Can you go deeper than root?

Reply:
I just moved those files and corrected the path. I could try `#SRC` though 🤔

Reply:
Please try it :)